tests/kuttl/common/scripts/delete_bmh.sh

```diff
@@ -4,11 +4,12 @@
 # Dumps a BMH and its credentials secret to designated locations and then deletes the BMH
 #
 
-bmhname=$(oc get -n openstack osbms compute -o json | jq -r '.status.baremetalHosts["compute-0"].hostRef')
-bmhsecretname=$(oc get bmh -n openshift-machine-api ${bmhname} -o json | jq -r '.spec.bmc.credentialsName')
-oc get -n openshift-machine-api bmh ${bmhname} -o json |\
+bmhname=$(oc get bmh -n openshift-machine-api -l osp-director.openstack.org/name=compute -l osp-director.openstack.org/namespace=openstack -o name | head -1)
+bmhsecretname=$(oc get -n openshift-machine-api ${bmhname} -o json | jq -r '.spec.bmc.credentialsName')
```

Comment:
We don't need to pick a specific host/BMH.  We just need to delete one of them, and so we now just choose the first one in the list.